knowledge-base/grid-scroll-specific-row.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Scroll to a specific row in the Kendo UI for Vue Native Grid
+description: Learn how scroll to a specific row in the Kendo UI for Vue Native Grid
+type: how-to
+page_title: How to scroll to a specific row in the Kendo UI for Vue Native Grid
+slug: grid-scroll-specific-row
+tags: grid, scroll, rows, row
```

Comment:
add the product here, like:
vue, grid, datagrid, scroll, scrollTo, specific, row